llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -236,7 +236,7 @@ std::string mangleFuncItanium(StringRef BaseName, const FunctionType *FT) {
   return Res;
 }
 
-MDNode *generateSpecConstDefaultValueMetadata(StringRef SymID, Value *Default) {
+MDNode *generateSpecConstDefaultValueMetadata(Value *Default) {
```

Comment:
Good catch, we can merge it as a separate `NFC` PR quicker, if you prefer.

Reply:
Done here: https://github.com/intel/llvm/pull/7992